clients/src/main/java/org/apache/kafka/common/metrics/JmxReporter.java

```diff
@@ -160,17 +160,30 @@ private void reregister(KafkaMbean mbean) {
         }
     }
 
-    private static class KafkaMbean implements DynamicMBean {
+    public int mbeansQty() {
```

Comment:
Maybe. Where can I see the above mentioned code conventions? Where is the violation?
Tried googling and found next guideline:
https://kafka.apache.org/coding-guide.html
Don't see the exact point which I've violated.

Tried searching the code  - don't see much `Qty`, `Num`, or `Size` appendixes, but found some `Count`... Is this what you've meant?